ui/src/test/AuditView/audit-rules.test.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
+import { AuditHosts } from '@app/AuditView/audit-hosts';
+import { MemoryRouter } from 'react-router';
+import { act } from 'react-dom/test-utils';
+import { IntlProvider } from 'react-intl';
+import { Route } from 'react-router-dom';
+import { mockApi } from '../__mocks__/baseApi';
+
+const ComponentWrapper = ({ children, initialEntries = ['/dashboard'] }) => (
```

Comment:
Maybe we should move this out into a test utils file since it seems to be used frequently.

Reply:
Will not change the tests as we will be moving to the monorepo